api/src/messaging/consumers.py

```diff
@@ -81,6 +82,35 @@ async def receive_message(self, text_data_json: dict) -> dict:
             "last_name": self.member["last_name"],
             "preferred_name": self.member["preferred_name"],
         }
+        content_type, content_object, object_id = (
+            text_data_json["content_type"],
+            None,
+            text_data_json["object_id"],
+        )
+
+        if content_type and object_id > 0:
+            try:
+                content_type = await sync_to_async(ContentType.objects.get)(
+                    model=content_type
+                )
+                content_type = content_type.model_class()
+                content_object = await sync_to_async(content_type.objects.get)(
+                    id=object_id
+                )
+            except Exception as e:
+                print(str(e))
+                return {
+                    "type": "send_message",
+                    "id": 0,
+                    "message_body": str(e),
+                    "creator": "",
+                    "previous_message": None,
+                    "datetime_created": "",
+                    "tagged_users": [],
+                    "content_type": "",
+                    "object_id": "",
+                }
```

Comment:
I'm not sure if we care about handling exceptions just yet. I think for now it's fine to just assume you're getting valid data in text_json. I'll add another story to handle exceptions in the chat consumer later.